sandbox-backend/src/main/resources/db/migration/V1_1_0__demo_migration.sql

```diff
@@ -5,4 +5,102 @@ CREATE TABLE IF NOT EXISTS `DEMO_TABLE`
     `SOME_FIELD` varchar(20)
```

Comment:
Let's change the name of init script, something like that:
`V0000000_initial.sql` or `V1_000000_initial.sql`